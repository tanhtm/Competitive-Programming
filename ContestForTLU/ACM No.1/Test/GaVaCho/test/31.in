23779968 72689634
