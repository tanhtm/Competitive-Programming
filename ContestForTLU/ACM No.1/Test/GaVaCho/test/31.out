11215119 12564849
